medusa/tv/series.py

```diff
@@ -1223,7 +1223,7 @@ def load_from_indexer(self, tvapi=None):
         self.classification = getattr(indexed_show, 'classification', 'Scripted')
         self.genre = getattr(indexed_show, 'genre', '')
         self.network = getattr(indexed_show, 'network', '')
-        self.runtime = int(getattr(indexed_show, 'runtime', 0))
+        self.runtime = int(getattr(indexed_show, 'runtime', 0) or 0)
```

Comment:
Using fake values for stuff (like 60) is a bad idea.  Personally I'd prefer None to 0 for a situation where the value is not present, but as a runtime shouldn't ever be 0, 0 vs None has little bearing here.